old/lib/LedgerSMB/Form.pm

```diff
@@ -599,8 +598,8 @@ qq|<meta http-equiv="content-type" content="text/html; charset=$self->{charset}"
        }
        var lsmbConfig = {dateformat: '$dformat'};
     </script>
-    <script type="text/javascript" language="JavaScript" src="$dojo_location/dojo/dojo.js"></script>
-    <script type="text/javascript" language="JavaScript" src="$dojo_location/lsmb/main.js"></script>
+    <script type="text/javascript" language="JavaScript" src="$LedgerSMB::Sysconfig::dojo_location/dojo/dojo.js"></script>
+    <script type="text/javascript" language="JavaScript" src="$LedgerSMB::Sysconfig::dojo_location/lsmb/main.js"></script>
```

Comment:
What's wrong with using `$dojo_location` here? We didn't change anything in the diff which would require the change.

Reply:
$dojo_location has been removed by this pr